src/main/kotlin/stringcalculator/InputConverter.kt

```diff
@@ -0,0 +1,32 @@
+package stringcalculator
+
+import stringcalculator.operations.Add
+import stringcalculator.operations.Divide
+import stringcalculator.operations.Multiply
+import stringcalculator.operations.Operation
+import stringcalculator.operations.Subtract
+
+class InputConverter {
+    fun extractInts(splitString: List<String>): ArrayDeque<Int> {
+        return ArrayDeque(splitString.filter { isNumeric(it) }.map { it.toInt() })
+    }
+
+    fun convertToOperations(splitString: List<String>): List<Operation> {
+        return splitString.filter { !isNumeric(it) }.map { convertToOperation(it) }
```

Comment:
```suggestion
        return splitString
            .filter { !isNumeric(it) }
            .map { convertToOperation(it) }
```
함수 체이닝 컨벤션을 적용해주시면 좋을 것 같아요

ref. [Coding conventions | Kotlin - Wrap chained calls﻿](https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls)